src/main/java/com/savvato/tribeapp/repositories/AdverbRepository.java

```diff
@@ -1,13 +1,17 @@
 package com.savvato.tribeapp.repositories;
 
 import com.savvato.tribeapp.entities.Adverb;
-import com.savvato.tribeapp.entities.Verb;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.Optional;
 
 @Repository
 public interface AdverbRepository extends CrudRepository<Adverb, Long> {
-    Optional<Verb> findByWord(String word);
+    Optional<Adverb> findByWord(String word);
+
+    @Query(nativeQuery = true, value = "select word from adverb where id = ?")
+    Optional<String> findAdverbById(Long id);
```

Comment:
This should be `findById(id)`... no need for the custom query that way.

Reply:
Reworking:

`Optional<Adverb> findById(id)` - returns an Adverb and requires a little extra code to get the string in the Phrase service.

`Optional<String> findWordById(id)` - seems to get the result I need, but that's different than what you asked for above and would mean making another findById(id) method for retrieving an adverb if/when we need it.

Which approach do you prefer?